html/modevlib/MozillaPrograms.js

```diff
@@ -23,12 +23,78 @@ var MozillaPrograms = {
 		// https://bugzilla.mozilla.org/rest/bug?include_fields=id,priority,product,component&chfield=[Bug%20creation]&product=Core&product=Firefox&product=Firefox%20for%20Android&product=Firefox%20for%20iOS&product=Toolkit&chfieldfrom=2016-06-01&chfieldto=NOW&f1=flagtypes.name&o1=notequals&resolution=---&v1=needinfo%3F&email1=intermittent-bug-filer%40mozilla.bugs&emailtype1=notequals&emailreporter1=1&o4=greaterthan&f4=bug_id&limit=10000&v4=0
 
 
-		["Firefox Triage", null, null, {
+		["Firefox Un-Triaged", null, null, {
 			"and": [
-				{"term": {"bug_status": "new"}},
+				{"not": {"term": {"bug_status": "resolved"}}},
 				{"terms": {"product": ["core", "firefox", "firefox for android", "firefox for ios", "toolkit"]}},
-				{"term": {"assigned_to": "nobody@mozilla.org"}},
-				{"range":{"created_ts":{"gte":Date.newInstance("2016-06-01").getMilli()}}},
+				// {"term": {"assigned_to": "nobody@mozilla.org"}},
+				{"range":{"created_ts":{"gte": new Date("2016-06-01").getMilliseconds()}}},
```

Comment:
The BZ ES cluster stores all times as numbers in milliseconds-since-epoch GMT.  `Date.newInstance("2016-06-01").getMilli()` returns the date's milliseconds since epoch.  `new Date("2016-06-01").getMilliseconds()` returns only the millisecond portion of time.  

Concerning `new Date("2016-06-01")`: In general it is dangerous to use any Javascript time or date function because you can not really predict what they will return; they depend on the running context; both in the browser and the os, giving you dates and times that are near-impossible to debug.  `Date.newInstance()` was designed to deliver times in GMT; it is context free function that is easier to reason about.